frontend/src/common/vaultconfig.ts

```diff
@@ -35,12 +37,14 @@ export class VaultConfig {
 
     const vaultConfigToken = await vaultKeys.createVaultConfig(kid, hubConfig, jwtPayload);
     const rootDirHash = await vaultKeys.hashDirectoryId('');
-    return new VaultConfig(vaultConfigToken, rootDirHash);
+    return new VaultConfig(vaultConfigToken, vaultUvf, rootDirHash);
   }
 
   public async exportTemplate(): Promise<Blob> {
     const zip = new JSZip();
+    // TODO https://github.com/encryption-alliance/unified-vault-format/pull/19 what about vault.uvf? Not in template but only in vault dto?
```

Comment:
@overheadhunter what about vault.uvf? Not in template but only in vault dto?

Reply:
The vault will be either v8 or uvf, not both at the same time. When I said Hub will support both in parallel, I wasn't refering to the same vaults. New vault will use uvf, while keeping older ones is still supported with the existing mechanisms.